### DISCUSSION TITLE
Autocompletion of tags in event names

### DISCUSSION THREAD

**Discussion by mkdjr** [davish/obsidian-full-calendar#418] [Feature Requests]
Would it be possible to have the input space for an event name have some of the same features as the editor in Obsidian? For example, autocompleting / suggesting tags and links in the event title input box. A larger space for input would also be welcome : )

Please let me know if there's anything I can do to help with development. This plugin has changed the way I work with Obsidian and I'd like to assist how I can.

**Comment by davish** [comment]
Improvements to the editor modal is one of the things I'm really considering after my current refactor in #303. I certainly appreciate the desire to help! If you're familiar with react and frontend development, working on #249 would be extremely helpful 😃

**Comment by mkdjr** [comment]
Hi there! I think I overcommitted when I offered to help with the plugin development as work is busy and I can't say I've ever worked in frontend development. So just letting you know I won't be able to contribute, and I thank you for your amazing work on the plugin : )

**Comment by mkdjr** [comment]
I asked about how to use obsidian's editor features in the prompt and depose#2272 provided the following helpful reply:

> I think you can extend `MarkdownEditView` https://github.com/obsidianmd/obsidian-api/blob/master/obsidian.d.ts#L1863

**Comment by mkdjr** [comment]
I saw that link auto completion was recently implemented in notion-like-tables and thought that might be related to this request, so I'm linking to that [PR](https://github.com/trey-wallis/obsidian-notion-like-tables/pull/483).
